test/biconnected_test.jl

```diff
@@ -1,12 +1,14 @@
+using LinearAlgebra
 @testset "biconnected" begin
     n = 10
     O = ones(Int64,n-1)
     Z = zeros(Int64,n)
     A = Tridiagonal(O,Z,O)
     B = MatrixNetwork(dropzeros!(sparse(A)))
     obj = biconnected_components(B)
-    i = 0
-    for i = 2:n
+    global i = 0
```

Comment:
Don't use global here...

Reply:
Add the @test on the last loop with the break

Reply:
Not sure if this is what you meant, but this is what I'm thinking for these for loops:
current code: https://github.com/nassarhuda/MatrixNetworks.jl/blob/df9fadf600619743f3cfe1e48c15fcaf0604b345/test/biconnected_test.jl#L9-L19

suggestion for new code:
```
    for i = 2:n
        @test obj.articulation_points[i]!=0
    end
```

Reply:
Good!

Reply:
Fixed here e78c50bb1ca0413f5207c6f904ee066672d4ee99